pkgs/tools/security/browserpass/default.nix

```diff
@@ -0,0 +1,34 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.0
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "browserpass-unstable-${version}";
+  version = "HEAD";
```

Comment:
Use proper way to name pkg. In this case it should be `browserpass-(date of commit)`. Also please use `sha` for consistency.